semantics.json

```diff
@@ -342,6 +342,14 @@
         "default": true
       },
       {
+        "name": "randomizeOnRetry",
+        "type": "boolean",
```

Comment:
Interesting question since MultiChoice didn't have that feature at all in the first place :-) I am not really sure. If not, why would there be an option to randomize the answers at all instead of always randomizing them right away?

I can at least imagine a situation where the order of questions is set deliberately and needn't be changed, e.g. scientific studies.